trademark.md

```diff
@@ -0,0 +1,72 @@
+Dask Trademark Policy
+=====================
+
+This document describes the policy governing the use of the "Dask" trademark,
+which is owned by NumFOCUS and administered on behalf of the Dask project.  Any
+use of the Dask trademark commercially or non-commercially must be in
+accordance with the policy described in this document unless specific written
+permission is obtained from NumFOCUS.
+
+Usage and modification of the Dask *software* is governed by the
+[license](https://github.com/dask/dask/blob/main/LICENSE.txt), which describes
+permissible behavior under copyright law.  Trademarks are a distinct form of
+intellectual property, and so the guidelines in this document apply only to
+the "Dask" trademark and logo, but not to the software itself.
+
+Intent
+------
+
+Our goal is to enable the Dask trademark and logo to be used with minimal
+restrictions by both commercial and non-commercial entities, but we do *not*
+want the Dask trademark to be used:
+
+* to cause confusion about whether a project, product, or service is endorsed
+  by the Dask community in some way.
+* in a way that allows an entity to misleadingly imply ownership of Dask or
+  influence over the Dask project.
+
+Permissible usage
+-----------------
+
+The Dask trademark is always subject to "nominative use rules" which allow the
+usage of the trademark for descriptive purposes that does not suggest
+sponsorship or endorsement by the trademark owner.  You may freely describe
+that your product or service "uses Dask" or is "powered by Dask," and you
+may use the unaltered Dask logo to indicate this without prior approval.
+(Please see the [Dask Brand Guide](https://www.dask.org/brand-guide) for more
+details about how to use the Dask logo.)
+
+Open source projects may also use the Dask name as part of the name of their
+project (Ex: `dask-cuda`) so long as it satisfies *all* the following
+criteria:
+
+* The project is open source with a liberal license.
+* The project is related to Dask.
+* The project name should not be overly grand in claiming namespace.
+* The project should not make subjective claims in the name itself.
+
+Usage of the "Dask" trademark should also be consistent with the [Dask Code of Conduct](https://github.com/dask/governance/blob/main/code-of-conduct.md).
+
+Non-permissible usage
+---------------------
+
+The Dask trademark may *not* be used in a commercial product or company name.  Examples of disallowed uses include:
+
+* Dask Pro
+* Dask Enterprise
+* The Dask Company
```

Comment:
@mrocklin I note that the Coiled website says "Coiled: A Dask Company" which to me seems to be 99% the same as this disallowed example.

Should this document be updated to allow for your use of this? Or are you planning on changing your strapline?

Reply:
Good question.  I think it's useful for consumers to know that we are a company pretty much entirely focused around Dask.  I'd say that today we are, de-facto, "The Dask Company" but aren't saying so, instead saying "A Dask Company" to make space potentially for others.  

What do folks think?

Reply:
Let's think about this a bit more.  I think that there are a few outcomes:

1.  We decide that "A Dask Company" in a tagline (rather than the company name) is relaxed and fine, like saying "NVIDIA Scale, powered by Dask"
2. We decide that "A Dask Company" in a tagline is not sufficiently relaxed, but that the trademark holder (NumFOCUS in the near future hopefully) grants a license for this use based on Coiled's current status as primary contributors.  If Coiled steps down from this status then NumFOCUS (or some Dask subcommittee within NumFOCUS) retracts this license.
3. We decide that no one should do this for any reason

Reply:
I see your intent, but I think a common interpretation of `A Dask Company` would be "A company that is owned by Dask".

Here are some real world examples of this usage:
- [Taos: an IBM company](https://www.taos.com) - Taos is owned by IBM
- [Ally.io, a Microsoft Company](https://twitter.com/allydotio?lang=en) - Ally.io was purchased by Microsoft
- [Verily: We are an Alphabet company focused on applying AI & data science](https://verily.com) - Verily is owned by Alphabet

Given this common interpretation I think that option 3 is the correct choice here. Coiled also uses the tagline "Dask that just works" which I think fits much better with the trademark policy here.

Reply:
>  I personally would also be up for discussing option 2, but would suggest strong consensus between all owners

I'd like to have this conversation.  I think that it benefits users and reflects reality a bit.  We happen to have a call in 20m.  I'd like to reserve 5m at the end for this if you're game.